odoo/addons/base/models/ir_model.py

```diff
@@ -1491,6 +1492,48 @@ def group_names_with_access(self, model_name, access_mode):
         """ % access_mode, [model_name])
         return [('%s/%s' % x) if x[0] else x[1] for x in self._cr.fetchall()]
 
+    @tools.ormcache('self.env.uid', 'self.env.su')
+    def _acl_map(self):
```

Comment:
What is the point of `ormcache`?  Currenlty this thing is only used to determine which menu entries are visible.  So it is used **once** for quite some time.  Okay, every time you refresh your page, which is... once a day?  Unless this method is used more frequently, there is no point in caching it.